app/src/components/Counter.tsx

```diff
@@ -0,0 +1,33 @@
+import type { PropsWithChildren } from 'react'
+//TODO change import to @hooks
+import useGetCounter from '../hooks/useGetCounter'
+//TODO check why this doesnt work with @styles
+import '../styles/index.css'
+
+interface Props {
+  seconds: number
+  isCountingUp?: boolean
```

Comment:
This does not need to be optional because it always has a default value, either `true` or `false`.